.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,4 +1,20 @@
-- [ ] Read the [theme publishing doc](https://hexo.io/docs/themes#Publishing) or [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
+## Check List
+
+**Please read and check followings before submitting a PR.**
+
+- [ ] I want to publish my theme on Hexo official website.
+    - [ ] I have already read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
```

Comment:
"I have read...", applies to the rest.

Reply:
Thanks :)
I will fix later. Also others.